tv_apps/smart-home/js/card_picker.js

```diff
@@ -37,7 +37,8 @@
       });
       this._keyNavigationAdapter.on('enter-keyup', this.onEnter.bind(this));
 
-      this._selectedElements = this.gridView.getElementsByClassName('selected');
+      this._selectedButtons = this.gridView.getElementsByClassName('selected');
+      this._appButtons = this.gridView.getElementsByTagName('smart-button');
 
       this.refresh();
```

Comment:
Not related to this bug, but ... Do we need to call `refresh` in the `init` stage? I suppose it should be called again when it's about to show, right?